src/index.html

```diff
@@ -22,6 +22,355 @@
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-counter">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-counter">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
```

Comment:
The 'href' attribute of the 'Buy' button is set to '#', which means it doesn't lead anywhere. Consider updating this to point to a relevant page or action.